lib/routes/typst/universe.ts

```diff
@@ -57,19 +57,22 @@ export const route: Route = {
             vm.runInContext(packages, context, {
                 displayErrors: true,
             });
-            const md = markdownit('commonmark');
-            const items = context.an.exports
-                .sort((a, b) => b.updatedAt - a.updatedAt)
-                .map((item) => ({
-                    title: `${item.name} | ${item.description}`,
-                    link: `https://typst.app/universe/package/${item.name}`,
-                    description: md.render(item.readme),
-                    pubDate: parseDate(item.updatedAt, 'X'),
+            const items = context.an.exports.sort((a, b) => a.updatedAt - b.updatedAt);
+            const groups = new Map(items.map((it) => [it.name, it]));
+            const pkgs = [...groups.values()].map(async (item) => await cache.tryGet(`typst:universe:${item.name}:${item.version}`, async () => {
+                    const pkgLink = `https://typst.app/universe/package/${item.name}`;
+                    const $ = load(await ofetch(pkgLink));
+                    return {
+                        title: `${item.name} (${item.version}) | ${item.description}`,
+                        link: pkgLink,
+                        description: $('section#readme').html(),
+                        pubDate: parseDate(item.updatedAt, 'X'),
+                    };
```

Comment:
This sends around 300 concurrent requests to the site, which is a big no-no.

You already have enough information to fix those broken images in each of `context.an.exports`:
- If an image URL starts with `https://github.com`, you need to rewrite it with `https://raw.githubusercontent.com`.
- If an image URL is a relative path, it means the image is stored in `https://github.com/typst/packages/tree/main/packages/preview`, and you need to rewrite the image URL with `https://raw.githubusercontent.com` as well.